---
layout: event-post
serie: Career Development Series
# categories: [blog]
name: Spatial transcriptomics application from bench to industry
tags: []
date-and-time: 2024-02-11, 1:00â€“2:30 PM ET
location: Lehman Hall 201 (8 Harvard Yard, Cambridge, MA)
organizers: ['BTBA']
attend: Register [here](https://forms.gle/AuxUhaW1q7wusrRW8)
fb-link: https://www.facebook.com/events/398146829541203/
---

éš¨è‘—æ¬¡ä¸–ä»£å®šåºçš„ç™¼å±•ï¼Œå„å¼å„æ¨£çš„é«”å­¸(omics)ä»¥åŠå…¶å–®ç´°èƒçš„è¡ç”ŸæŠ€è¡“å¤§å¤§æ“´å±•äº†æˆ‘å€‘å°åŸºå› èª¿æ§èˆ‡åˆ†å­æ©Ÿåˆ¶çš„ç†è§£ï¼Œç„¶è€Œå…¶æŠ€è¡“ä¸Šæœ€å¤§çš„ç¼ºé»æ˜¯ä»–æµå¤±äº†åŸæœ‰ç´°èƒæˆ–çµ„ç¹”çš„ç©ºé–“è¨Šæ¯ã€‚éå»åå¹´ï¼Œç©ºé–“çµ„åˆ†æ(spatial omics)çš„å¿«é€Ÿé€²å±•çªç ´äº†é€™å€‹é™åˆ¶ï¼Œè®“æˆ‘å€‘å¾—ä»¥ç ”ç©¶å®Œæ•´ç´°èƒæˆ–çµ„ç¹”å…§åŸºå› è¡¨é”åŠå…¶èª¿æ§åœ¨ç©ºé–“ä¸Šçš„é—œä¿‚ã€‚

åœ¨é€™å€‹æœˆæœƒè£¡ï¼Œæˆ‘å€‘å¾ˆæ¦®å¹¸åœ°é‚€è«‹åˆ°ä¸‰ä½ä¾†è‡ªä¸åŒé¢å‘çš„è¬›è€…ä¾†ç‚ºæˆ‘å€‘ä»‹ç´¹å–®ç´°èƒç©ºé–“è½‰éŒ„çµ„åˆ†æåœ¨åŸºç¤ç§‘å­¸ä¸Šçš„æ‡‰ç”¨ï¼Œç©ºé–“çµ„é«”å­¸æŠ€è¡“å…¬å¸çš„é–‹ç™¼ç¾æ³èˆ‡å‰æ™¯ï¼Œèˆ‡åœ¨è—¥å» è£¡æ–°è—¥é–‹ç™¼ä¸Šçš„å¹«åŠ©ã€‚é€™å ´æ¼”è¬›å°‡æä¾›ä¸€å€‹å…¨é¢çš„è¦–è§’ï¼Œæ¢è¨å–®ç´°èƒç©ºé–“è½‰éŒ„çµ„åˆ†æçš„å‰æ²¿æ‡‰ç”¨ï¼Œä»¥åŠå®ƒåœ¨ä¸åŒé ˜åŸŸä¸­æ‰€å¸¶ä¾†çš„é‡è¦çªç ´ã€‚æˆ‘å€‘æœŸå¾…è‘—èˆ‡å¤§å®¶åˆ†äº«é€™äº›ä»¤äººæŒ¯å¥®çš„ç™¼ç¾å’ŒæŠ€è¡“é€²å±•ã€‚

In the past decade, substantial progress has been made in bulk omics technologies and their single-cell derivatives, enhancing our understanding of biology and disease mechanisms. However, an unsolved limitation is the loss of spatial information inherent in these techniques, hindering a comprehensive portrayal of fundamental biological processes. To address this, spatial omics have rapidly emerged as a transformative technique, allowing measurements of the spatial distribution of molecules in cells and tissues and thus unlocking the spatial patterns of global gene expression.

In this BTBA meeting, we are honored to have three speakers from different perspectives to introduce the applications of single-cell spatial transcriptomics in basic science, the current developments and prospects specializing in spatial omics technologies company, and its role in drug development. This presentation will offer a comprehensive perspective, exploring the cutting-edge applications of single-cell spatial transcriptomics and the significant breakthroughs it brings to different fields. We look forward to sharing these exciting discoveries and technological advancements with everyone.

ğŸ—£ï¸ æœ¬æ´»å‹•å…¨ç¨‹ä»¥ä¸­æ–‡é€²è¡Œ. The event will be in Mandarin; hybrid event

â–¶ï¸ ä¸»é¡Œ / Topic: Spatial transcriptomics application from bench to industry

â–¶ï¸ æ™‚é–“ / Time: 2024-02-11 (Sun)
- 12:30 pm - 01:00 pm (ET) Networking
- 01:00 pm - 2:30 pm (ET) Spatial transcriptomics Panel
- 02:30 pm - 03:00 pm (ET) Networking
- 03:00 pm - 04:00 pm (ET) BTBA monthly meeting
- 04:00 pm - 05:00 pm (ET) Lunar New Year Party (in-person only)

â–¶ï¸ æ´»å‹•åœ°é»/Venue: Lehman Hall 201 (8 Harvard Yard, Cambridge, MA) & online link (è«‹å¡«å¯«å ±åè¡¨ï¼Œæœƒè­°é€£çµè¬›æœƒåœ¨æ´»å‹•å¹¾å¤©å‰å¯„å‡ºï¼‰

â–¶ï¸ å ±åé€£çµ/RSVP : <https://forms.gle/wXLM5DcMuEehQhZ49>

â–¶ï¸ è¬›è€… Speakers:

- I-Hao Wang ç‹ç¿Šè±ª
- Cheng-Yi Chen é™³æ”¿å„€
- Shih-Ying Wu å³è©©ç›ˆ

â–¶ï¸ ä¸»æŒäºº Moderators:

- Chun-Chen Jerry Yao å§šä¿Šè¾°


### Speakers

#### I-Hao Wang ç‹ç¿Šè±ª
PhD candidate, UMass Chan Medical School

I-Hao, a neuroscience Ph.D. candidate at UMass Chan Medical School, specializes in unraveling the complexities of olfactory neural circuitry in rodents. His doctoral research delves into the intricate wiring of these neural networks and examines how their structural design aids in processing external stimuli. Utilizing advanced spatial single-cell genomics, I-Hao identifies pivotal transcriptional signatures in olfactory sensory neurons. He then employs machine learning techniques to construct a detailed axon projection map within the brain. By analyzing the relationship between this map and sensory responses, I-Hao's work illuminates the process by which neuronal circuitry translates varied chemical structures into distinct patterns of neuronal activation.

I-Hao is currently exploring opportunities in the job market and is eager to leverage his expertise in both wet and dry lab methodologies in future research endeavors. He welcomes connections and is open to discussing potential collaborations or roles. Please don't hesitate to reach out!

LinkedIn page: <https://www.linkedin.com/in/i-hao-wang-693934196/>


#### Cheng-Yi Chen é™³æ”¿å„€
Bioinformatics Scientist at Vizgen, Boston, MA

Cheng-Yi Chen earned his Ph.D. in Developmental Biology from the Stowers Institute for Medical Research in Kansas City, MO. His doctoral research centered on germline stem cell development and evolution. In the latter part of his Ph.D., Cheng-Yi developed a keen interest in big data analysis, prompting a career shift towards a hybrid of wet and dry lab methodologies. This led him to postdoctoral research in zebrafish transcriptomics at the NICHD/NIH, where he mastered the entire single-cell RNA sequencing workflow, from library preparation to bioinformatics data analysis.

After turning down the NIH Fi2 grant, an intramural counterpart of the F32 grant, Cheng-Yi joined Vizgen as a Bioinformatics Scientist. In this role, he contributes to customer project design, data quality control and management, R&D software development, and spatial single-cell genomics analysis. With extensive experience in molecular biology, developmental biology, evolution, genetics, genomics, and bioinformatics, Cheng-Yi is passionate about sharing his journey from benchwork to computational roles and his transition from academia to industry.

LinkedIn page: <https://www.linkedin.com/in/penguinayee/>


#### Shih-Ying Wu å³è©©ç›ˆ
Senior Advisor at Eli Lilly and Company, Cambridge, MA

Shih-Ying is a senior advisor in Data Science and Bioinformatics for Next Generation Therapeutics in Lilly Neuroscience, passionate about digital innovation to improve human health. As a computational scientist, she leads projects to support drug development pipelines for neurodegenerative diseases and pain, including transcriptomics, spatial omics, etc. Before joining Lilly, she served as a senior bioinformatics specialist in the Research Computing Group in the IT department at Boston Childrenâ€™s Hospital, leading bioinformatics projects in collaboration with research labs and investigators from design, pipeline building, data analysis, to interpretation, including whole exon/genome sequencing for patient cohorts, single-cell RNA sequencing to study disease-relevant targets and cell types. She earned her Ph.D. in Biomedical Engineering with honors at Columbia University for drug delivery to the brain to treat neurodegenerative diseases, a master's degree in Biomedical Electronics and Bioinformatics at National Taiwan University, and a bachelor's degree in Electrical Engineering at National Tsing Hua University in Taiwan. She has over 18 peer-reviewed journal publications and recognitions including the Lilly Research Laboratory Presidentâ€™s Award. In her leisure time, she enjoys quality time with her family, reading, exploring new places, and learning new skills. She also actively promotes an inclusive and supportive culture.

LinkedIn page: <https://www.linkedin.com/in/shih-ying-wu>


### Moderator


#### Chun-Chen Jerry Yao å§šä¿Šè¾°
Ph.D. student in Dana-Farber Cancer Institute and Department of Molecular and Cellular Biology, Harvard University

Jerry is a graduate student in Harvardâ€™s Molecules, Cells, and Organisms (MCO) program. He earned his BSc at MIT in Brain and Cognitive Science (BCS), where he optimized an in situ sequencing-by-synthesis method in Ed Boydenâ€™s lab and Fei Chenâ€™s lab; he also studied transcription-translation uncoupling in Bacillus subtilis in Gene-Wei Liâ€™s lab. During his gap year before starting at Harvard, he served the Taiwanese army and then worked with Dan Tawfik at the Weizmann Institute to investigate the origin of peptides and the genetic code in the origin of life. For his Ph.D., he hopes to elucidate the principles of biomolecular evolution of enzyme specificity using protein design, evolutionary biology, and high-throughput experimental approaches.
